src/gui/advancedsettings.cpp

```diff
@@ -459,7 +462,13 @@ void AdvancedSettings::loadAdvancedSettings()
     addRow(HASHING_THREADS, (tr("Hashing threads") + ' ' + makeLink("https://www.libtorrent.org/reference-Settings.html#hashing_threads", "(?)"))
             , &m_spinBoxHashingThreads);
 #endif
-
+    // Max active rechecking torrents
+    m_spinBoxMaxActiveCheckingTorrents.setMinimum(-1);
+    m_spinBoxMaxActiveCheckingTorrents.setMaximum(std::numeric_limits<int>::max());
+    m_spinBoxMaxActiveCheckingTorrents.setValue(session->maxActiveCheckingTorrents());
+    m_spinBoxMaxActiveCheckingTorrents.setSpecialValueText(QString::fromUtf8(C_INFINITY));
+    addRow(MAX_ACTIVE_CHECKING_TORRENTS, (tr("Maximum active checking torrents") + ' ' + makeLink("https://www.libtorrent.org/reference-Settings.html#active_checking", "(?)"))
```

Comment:
Maybe it is better to add the option under "Torrent Queueing"?
This specific option only has effect when auto manage is ON and "Torrent Queueing" is where the related options are placed.

Reply:
I see. But that would require some work in the .ui files. I do not know how to do that.

Reply:
This would cause other inconveniences: Almost everyone, always would want this option enabled so the groupbox checkbox "Torrent Queueing" would always be enabled thus making it redundant. But if someone only wants the max active checking torrents option enabled, he'd have to set everything else there to -1. Unless of course a checkbox for each of them is re-introduced in order to remember the last used value.

Reply:
OK, forget it.

But is there somewhere that it can reside? I feel like this is a commonly used option that shouldn't belong in Advanced settings group. Maybe just somewhere under BitTorrent tab?

>I see. But that would require some work in the .ui files. I do not know how to do that.

I can help with it. I'll take this PR and add my changes (also add myself as second author, since you are the first author) and propose another PR.

Reply:
> propose another PR.

You can push to this branch, can't you?

Reply:
>You can push to this branch, can't you?

I realize it is not so good to mess up the original PR branch.

Reply:
See PR #16584.